workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx

```diff
@@ -87,27 +87,27 @@ const WorkflowDefinitionDetailsCard = ({
   return (
     <InfoCard title="Details" className={classes.details}>
       <Grid container spacing={3} alignContent="flex-start">
-        <Grid container item md={4} spacing={3} alignContent="flex-start">
+        <Grid container item md={12} spacing={3} alignContent="flex-start">
           {details?.map(({ label, value, children }) => (
-            <Grid item md={6} key={label}>
+            <Grid item md={3} key={label}>
               {/* AboutField requires the value to be defined as a prop as well */}
               <AboutField label={label} value={value}>
                 {loading ? <Skeleton variant="text" /> : children || value}
               </AboutField>
             </Grid>
           ))}
-        </Grid>
-        <Grid item md={8}>
-          <AboutField
-            label="description"
-            value={formattedWorkflowOverview?.description}
-          >
-            {loading ? (
-              <Skeleton variant="text" />
-            ) : (
-              formattedWorkflowOverview?.description
-            )}
-          </AboutField>
+          <Grid item md={3}>
+            <AboutField
+              label="description"
+              value={formattedWorkflowOverview?.description}
+            >
+              {loading ? (
+                <Skeleton variant="text" />
+              ) : (
+                formattedWorkflowOverview?.description
+              )}
+            </AboutField>
+          </Grid>
```

Comment:
are these changes needed?

Reply:
@gciavarrini 
WDYM? How would you make the change in the screen otherwize?

Reply:
@batzionb 
I'm ok with the changes, but it seems that those lines were just indented differently. That's why I'm asking if it was intended. 
Not a "blocker" comment :)

Reply:
if you look carefully you see it's not just an indent change
The </Grid> close tag was moved down

Reply:
ohh! ok now i see it. Thanks